src/Uno.UI/Controls/BindableNSView.macOS.cs

```diff
@@ -59,7 +59,7 @@ public BindableNSView()
 			}
 		}
 
-		public BindableNSView(IntPtr handle)
+		public BindableNSView(ObjCRuntime.NativeHandle handle)
```

Comment:
Does this need to remain compatible with the legacy SDK ? If so

```csharp
using ObjCRuntime;
#if !NET
using NativeHandle = System.IntPtr;
#endif
```
and use `NativeHandle` elsewhere in the file

Reply:
Indeed, fixed in a later change. CI was not happy about it ;)